pkg/chunk/chunk.go

```diff
@@ -327,3 +329,47 @@ func (c *Chunk) Samples(from, through model.Time) ([]model.SamplePair, error) {
 	interval := metric.Interval{OldestInclusive: from, NewestInclusive: through}
 	return prom_chunk.RangeValues(it, interval)
 }
+
+// Slice builds a new smaller chunk with data only from given time range
+func (c *Chunk) Slice(from, through model.Time) (*Chunk, error) {
```

Comment:
If I'm not wrong, the `Chunk.From` and `Chunk.Through` are both inclusive. If confirmed, it may make sense having the input `from` and `through` inclusive as well for consistency (and whatever decision we pick about it we should document it in the function).

Reply:
Yes, the time ranges are inclusive in both cases. I will add a comment.
Thanks!